src/lib/components/workspace/team-invite/TeamInvite.svelte

```diff
@@ -268,12 +294,43 @@
     <p class="invite-subheader text-textColor mt-0 mb-1">
       Select workspaces you would want to give access to.
     </p>
-    <CheckSelectDropdown
+    <!-- <CheckSelectDropdown
       isError={workspaceError && !countCheckedList(teamSpecificWorkspace)}
       id={"check-select-workspace"}
       list={teamSpecificWorkspace}
       onclick={handleCheckSelectDropdown}
-    />
+    /> -->
+    <Dropdown
+      dropDownType={{ type: "checkbox", title: "select" }}
+      dropdownId="check-select-workspace"
+      data={[
+        {
+          name: "Select",
+          id: "select",
+          dynamicClasses: "text-whiteColor",
+          hide: true,
+        },
+        {
+          name: "Select ALL",
+          id: "select-all",
+          dynamicClasses: "text-whiteColor",
+          isInvalidOption: true,
+          checked: isAllSelectedCheck,
+        },
```

Comment:
if this would be same for all, move it to constant and import and use it here with ... operator